modules/util/util.go

```diff
@@ -25,6 +26,20 @@ const (
 	OptionalBoolFalse
 )
 
+var matchComplexity *regexp.Regexp
+var matchComplexityOnce sync.Once
+
+// CheckPasswordComplexity return True if password is Complexity
+func CheckPasswordComplexity(pwd string) bool {
+	matchComplexityOnce.Do(func() {
+		matchComplexity = regexp.MustCompile(setting.PasswordComplexity)
+	})
+	if matchComplexity.MatchString(pwd) {
```

Comment:
Can drop this if/else to `return matchComplexity.MatchString(pwd)`

Reply:
Yes, You are right